WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/viewholders/SiteViewHolder.kt

```diff
@@ -1,18 +1,32 @@
 package org.wordpress.android.ui.reader.subfilter.viewholders
 
+import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
 import org.wordpress.android.R
 import org.wordpress.android.ui.reader.subfilter.SubfilterListItem.Site
 import org.wordpress.android.ui.utils.UiHelpers
+import org.wordpress.android.util.UrlUtils
 
 class SiteViewHolder(
     parent: ViewGroup
 ) : SubfilterListItemViewHolder(parent, R.layout.subfilter_list_item) {
     private val itemTitle = itemView.findViewById<TextView>(R.id.item_title)
+    private val itemUrl = itemView.findViewById<TextView>(R.id.item_url)
 
     fun bind(site: Site, uiHelpers: UiHelpers) {
         super.bind(site, uiHelpers)
         this.itemTitle.text = uiHelpers.getTextOfUiString(parent.context, site.label)
+        this.itemUrl.visibility = View.VISIBLE
+
+        val blog = site.blog
+
+        this.itemUrl.text = if (blog.hasUrl()) {
```

Comment:
Maybe we can switch to `when` for better readability here?

Reply:
TIL! I was actually not aware of this when without parameter form 😬, thanks 😊! Changed here f97bbf90c05ca39e67825776a94c71d15142de1c